src/olympia/reviewers/utils.py

```diff
@@ -808,10 +808,17 @@ def get_actions(self):
         )
 
     def process(self):
-        action = self.handler.data.get('action', '')
-        if not action:
+        requested_action = self.handler.data.get('action', '')
+        if not requested_action or requested_action not in self.actions:
```

Comment:
depending on your love for walruses, 
```suggestion
        if not (action := self.actions.get(self.handler.data.get('action'))):
```
(and replaces `action = self.actions[requested_action]` too)

Reply:
I like that, 849a04ab2d717a3c6b0fa445e1ab2a078286ff36